### ISSUE TITLE
:Calendar restarts authorization after I paste Googles code when using neovim

### ISSUE THREAD

**Issue by JJJHolscher** [itchyny/calendar.vim#214]
The final step in https://github.com/itchyny/calendar.vim#important-notice re-opens the browser and again requests I put in the code that I get from there.
My API key works, I tested it with gcal-cli afterwards.

The prompt for the request for pasting the code after pasting the code looks like:
```
Access https://accounts.google.com/o/oauth2/auth?client_id=$CLIENT_ID&redirect_uri=$LOCALHOST_UL&response_type=code&scope=$SCOPE and paste the code
CODE: ${$PASTED_CODE}Access https://accounts.google.com/o/oauth2/auth?client_id=$CLIENT_ID&redirect_uri=$LOCALHOST_UL&response_type=code&scope=$SCOPE and paste the code
CODE: 
```
Here I omitted some $VARIABLES
After re-entering the new code the whole process just repeats again.
If I open a new neovim instance and call :Calendar, I again go through this process.

**Comment by itchyny** [comment]
Start the calendar with `-debug` option and investigate the responses in `~/.vim/cache/calendar.vim/download/`.

**Comment by itchyny** [comment]
Sorry the default download directory is `~/.cache/calendar.vim/download`. Any case, could you debug with `echom content` [here](https://github.com/itchyny/calendar.vim/blob/8d4ab7a8d9120a3538216056047d10df3947789b/autoload/calendar/google/client.vim#L66)?

**Comment by JJJHolscher** [comment]
Actually I'm noticing gcalcli errors on me if I let it query all my calendars (it lists calendars I've removed). I'll generate a new API key and see if the problem persists. (if it does I'll donate as an apology).

**Comment by JJJHolscher** [comment]
I remade my API keys to be sure. Both gcalcli and the python library gcsa can work with it.
I tested it with a different account and the different account again works with gcalcli and gcsa but gets calendar.vim to give the same behavior as before.
I uninstalled all my neovim plugins to no avail (I use packer).
Removing `let g:calendar_google_calendar = 1` does get me an empty calendar.

What _did_ work was using vim instead of neovim. Vim was vanilla aside from cloning this repo into `~/.vim/pack/vendor/start`, sourcing the credentials and setting `g:calendar_google_calendar`. I did not need to log in even (It probably reuses the login from nvim).
